setup.py

```diff
@@ -1,20 +1,32 @@
 # Copyright (c) 2019 Radio Astronomy Software Group
 # Licensed under the 2-clause BSD License
 
-import glob
-import io
-import sys
+import glob  # pragma: nocover
+import io  # pragma: nocover
 
-from setuptools import setup
+from setuptools import find_packages, setup  # pragma: nocover
 
-# add pyradiosky to our path in order to use the branch_scheme function
-sys.path.append("pyradiosky")
-from branch_scheme import branch_scheme  # noqa
 
-with io.open("README.md", "r", encoding="utf-8") as readme_file:
+# define the branch scheme. Have to do it here so we don't have to modify the path
+def branch_scheme(version):  # pragma: nocover
+    """
+    Local version scheme that adds the branch name for absolute reproducibility.
+
+    If and when this is added to setuptools_scm this function can be removed.
+    """
+    if version.exact or version.node is None:
+        return version.format_choice("", "+d{time:{time_format}}", time_format="%Y%m%d")
+    else:
+        if version.branch == "main":
+            return version.format_choice("+{node}", "+{node}.dirty")
+        else:
+            return version.format_choice("+{node}.{branch}", "+{node}.{branch}.dirty")
+
+
+with io.open("README.md", "r", encoding="utf-8") as readme_file:  # pragma: nocover
```

Comment:
I wonder if we could just publish a small package with this branch scheme in it? Then require it in `pyproject.toml`?